src/standardMethods.ts

```diff
@@ -190,19 +187,26 @@ export function generateSimpleListEndpoint<RequestParams, ResponseType>(
 export function generateRetrieveEndpoint<IdType, ResponseType>(
   axiosInstance: AxiosInstance,
   resourcePath: string,
-  metadataMap: MetadataMap
+  metadataMap: MetadataMap,
+  chunkFunction?: (ids: IdType[]) => IdType[][]
 ) {
   return async function retrieve(ids: IdType[]): Promise<ResponseType[]> {
-    const response = await rawRequest<ItemsResponse<ResponseType>>(
-      axiosInstance,
-      {
-        method: 'post',
-        url: `${resourcePath}/byids`,
-        data: { items: ids },
-      },
+    const chunks = doChunking<IdType>(ids, chunkFunction);
+    const responses = await promiseAllWithData(
+      chunks,
+      input =>
+        rawRequest<ItemsResponse<ResponseType>>(axiosInstance, {
+          method: 'post',
+          url: `${resourcePath}/byids`,
+          data: { items: input },
+        }),
       true
```

Comment:
```suggestion
      false
```
runInSequence - false, we don't care about order of our requests here

Reply:
What if we want to keep the order? If you request [a,b,c] and you end up with [b,c,a] then thats not good.

Reply:
answered below

Reply:
@eirikdahlen Ok, it should be `false`